src/main/java/com/example/repository_details/model/Repository.java

```diff
@@ -0,0 +1,18 @@
+package com.example.repository_details.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.Setter;
+
+import java.time.LocalDateTime;
+
+@AllArgsConstructor
+@Getter
+@Setter
+public class Repository {
+    private String full_name;
```

Comment:
@JsonProperty("full_name") <- wskazujemy ze klucz w jsonie reprezentujacy to pole ma nazwe "full_name"
private String fullName;